sentinel/src/androidTest/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    package="com.infinum.sentinel"
```

Comment:
Package name inside of AndroidManifest is not supported anymore and we were getting errors/warnings, don't remember.

Reply:
We can set the `namespace` in build.gradle files.

Reply:
I think you don't need to set namespace for tests, gradle will just append .test by default https://developer.android.com/build/configure-app-module#change-namespace-for-testing